Kernel/Devices/FUSEDevice.cpp

```diff
@@ -27,37 +27,33 @@ UNMAP_AFTER_INIT FUSEDevice::~FUSEDevice() = default;
 
 ErrorOr<void> FUSEDevice::initialize_instance(OpenFileDescription const& description)
 {
-    return m_instances.with([&](auto& instances) -> ErrorOr<void> {
-        VERIFY(!instances.contains(&description));
-        TRY(instances.try_set(&description, {}));
-        return {};
-    });
+    SpinlockLocker locker(m_lock);
```

Comment:
Not as important, as the commit body explains it a bit, but "improve XXX" is not liked to be seen as a commit title